.github/workflows/quality-check.yml

```
@@ -0,0 +1,34 @@
+      - synchronize
+      - ready_for_review
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 22.14.0
+  PNPM_VERSION: 9.15.7
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    needs: []
```

actionlint (1.7.4)

"needs" section should not be empty

(syntax-check)